docs/DESIGN-SYSTEM.md

```
@@ -0,0 +1,534 @@
+```css
+:root {
+  --animation-duration: 250ms;
+  --reduced-motion-duration: 0.01ms;
+}
+```
+
+#### How It Works
+
+The design system provides two approaches for respecting reduced motion preferences:
+
+**1. Using CSS Variables (Recommended)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)